### DISCUSSION TITLE
Export single and selected records

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.Forms.Issues#1514] [Ideas]
Currently from record page there is no action to export record data.

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/2919859/f8396b0e-5a71-4294-9caf-1da9b3cd6e90)

It can be done from list view to export all. However to export a single record one has to filter the list to match the specific record and export all (filtered results).

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/2919859/f46d9429-98b0-47ba-9845-4412e9ec2ec8)

It would be great if single/multi selection in list view allowed one to export the selected records though, but currently this only enable deletion of the selected records.

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/2919859/f4c6efbc-35d3-48e9-ac8f-87cc8515f749)

**Comment by TvGessel** [comment]
We've added this to our backlog for refinement. We'll be looking at this in conjunction with other entity action improvements.
